hugo/data/projects.yml

```diff
@@ -1,2 +1,8 @@
 title: projects
 desc: "source{d} is building the open-source components that enable matchine learning on source code. We believe that the next step in programming comes from teaching machines to understand and write code."
+posts:
+  display: true
+  categories:
+    - category: technical
```

Comment:
already defined at `data/home/home.yml`; the blog part could be moved to `data/blog.yml`